### ISSUE TITLE
Warlock talent (Siphon Soul) doesn't work

### ISSUE THREAD

**Issue by Nonamesleftffs** [Looking4Group/L4G_Core#3757]
Talent reads

"Increases the amount drained by your Drain Life spell by an additional 4% for each Affliction effect on the target, up to a maximum of 60% additional effect. "

I believe that this talent doesn't work, i drain the same amount of HP with 0 Affliction effects on the target as i do with 15 Affliction effects on the target.

Any amount of testing performed will show this.

**Comment by Boonga** [comment]
There isn't 15 affliction spells that you could possibly have active all at the same time

**Comment by Nonamesleftffs** [comment]
Yes there is, you can have 15 warlocks cast 15 separate Corruptions... They're all Affliction spells, that's the point of the talent.

**Comment by Boonga** [comment]
It's per target tho u can't dot 10 mobs with 1 corruption then hope to get the 10 dot bonus on the 1 mob you siphon
